# Gamepad status messages

# Max value
int32 max

# Joysticks
int32 left_stick_x
int32 left_stick_y
int32 right_stick_x
int32 right_stick_y

# Triggers
int32 left_trigger
int32 right_trigger

# Dpad
bool dpad_left
bool dpad_right
bool dpad_up
bool dpad_down

# Buttons
bool A
bool B
bool X
bool Y
bool left_bumper
bool right_bumper
bool left_extra
bool right_extra
bool xbox
bool left_thumb
bool right_thumb
